1_preprocessing_data/PyBaSiC_Pipelines/scripts/nbconverted/Illumination_Correction.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Illumination Correction of NF1 Images
+
+# ## Import libraries
+
+# In[1]:
+
+
+import sys
+import numpy as np
+import matplotlib
+import pathlib
+from matplotlib import pyplot as plt
+from pathlib import Path
+import os
+import skimage
+
+# explicit import to PyBaSiC due to not having package support
+sys.path.append("./PyBaSiC/")
+import pybasic
+
+
+# ## Create path and load in images
+# The directory set up used for this experiment goes as such:
+# ```
+# 
+# ├── NF1_SchwannCell_data/
+# │   ├── 1_preprocessing_data/
+# │   │   ├── PyBaSiC_Pipelines/
+# │   │   │   ├── NF1_Channels/
+# │   │   │   │   ├── DAPI/ or GFP/ or RFP/
+# 
+# ```
+
+# In[2]:
+
+
+channel_path = '../PyBaSiC_Pipelines/NF1_Channels/RFP/'
```

Comment:
```suggestion
channel_path = pathlib.Path('../PyBaSiC_Pipelines/NF1_Channels')
channels = ["RFP", "GFP," "DAPI"]
```

There are two suggestions baked into this one suggestion.

1. When we're specifying a path variable, it is best practice to use the [`pathlib`](https://docs.python.org/3/library/pathlib.html) package.
2. Let's separate the channels variable, so that we can iterate over it